src/parser/ts/parser.unit.test.ts

```diff
@@ -250,6 +250,27 @@ describe('parse & reverse', () => {
   );
 });
 
+describe('escape characters parsing', () => {
+  const inputs = [
+    String.raw`ts{name=~"\d some"}`,
+    String.raw`ts{name=~"\\d some"}`,
+    String.raw`ts{name=~"\\d \" some"}`,
+    String.raw`ts{name=~"\\d \\" some"}`,
+  ];
+  const outputs = [
+    STS([Filter('name', String.raw`\d some`, RegexEquals)]),
+    STS([Filter('name', String.raw`\\d some`, RegexEquals)]),
+    STS([Filter('name', String.raw`\\d " some`, RegexEquals)]),
+    STS([Filter('name', String.raw`\\d \" some`, RegexEquals)]),
```

Comment:
```suggestion
    STS([Filter('name', String.raw`\\d \" some`)]),
```
it's a default value